src/hardware/steering_actuator/src/node_steering_actuator.cpp

```diff
@@ -389,7 +397,7 @@ class SteeringActuator : public rclcpp::Node, public CanInterface {
     void as_state_callback(const driverless_msgs::msg::State msg) {
         this->state = msg;
         if (msg.state == driverless_msgs::msg::State::DRIVING ||
-            msg.state == driverless_msgs::msg::State::ACTIVATE_EBS ||
+            msg.state == driverless_msgs::msg::State::ACTIVATE_EBS || msg.state == 105 ||
```

Comment:
This won't break things long term, but should probably change this from a hardcoded number before pushing to main, otherwise it may be forgotten

Reply:
I agree, turns out this fix doesnt work anyway because of steering wheel states